openapi/azure.iotcentral/client.bal

```diff
@@ -67,16 +65,17 @@ public isolated client class Client {
     public isolated function init(ClientConfig clientConfig, string serviceUrl = "/api") returns error? {
```

Comment:
Shall we add a valid default service URL here

Reply:
This URL is user specific. So we couldn't give a valid URL.

Reply:
We have to remove entire `server` section from the openapi. Since this is not a valid URL, it will be misleading for the user

Reply:
https://github.com/ballerina-platform/openapi-connectors/blob/797fdff3b2800c77136efeb5826353de8fc88146/openapi/azure.iotcentral/openapi.yaml#L16

Then let's remove the default value by removing these lines from the openapi.yaml

Reply:
Done.